### ISSUE TITLE
Train a model to reproduce domain-shift results

### ISSUE THREAD

**Issue by MosyMosy** [mboudiaf/TIM#9]
Hi, when I want to evaluate Domain shift (Table 2. in paper), it is needed to provide an appropriate pre-trained model (for example "checkpoints/mini2cub/softmax/resnet18") but, if we want to train the appropriate model from scratch, how it is possible to train it for the cross domain? 
I did not find the script.

**Comment by mboudiaf** [comment]
Hi,
Actually there is no need to have a specific mini2cub folder, as the model in this folder is really just a copy of the one trained on mini. So if you want to train from scratch, you can simply train on mini directly, and then set `ckpt_path="checkpoints/mini/softmax/resnet18"` in `mini2cub.sh`. I will update the script, thanks for notifying me.
Best

**Comment by MosyMosy** [comment]
Thank you for your response.
